pkg/provider/kubernetes/crd/kubernetes.go

```diff
@@ -58,8 +58,8 @@ type Provider struct {
 	IngressClass              string          `description:"Value of kubernetes.io/ingress.class annotation to watch for." json:"ingressClass,omitempty" toml:"ingressClass,omitempty" yaml:"ingressClass,omitempty" export:"true"`
 	ThrottleDuration          ptypes.Duration `description:"Ingress refresh throttle duration" json:"throttleDuration,omitempty" toml:"throttleDuration,omitempty" yaml:"throttleDuration,omitempty" export:"true"`
 	AllowEmptyServices        bool            `description:"Allow the creation of services without endpoints." json:"allowEmptyServices,omitempty" toml:"allowEmptyServices,omitempty" yaml:"allowEmptyServices,omitempty" export:"true"`
-
-	lastConfiguration safe.Safe
+	UseNativeLoadBalancer     bool            `description:"Use native Kubernetes Loadbalancing instead of traefik provided" json:"useNativeLoadBalancer,omitempty" toml:"useNativeLoadBalancer,omitempty" yaml:"useNativeLoadBalancer,omitempty" export:"true"`
```

Comment:
WDYT to rename the option to `NativeLBByDefault`?

Reply:
I have renamed this to `UseNativeLB`. I believe this makes more sense, but please let me know if you have any preferences, and I can make the change accordingly.

Reply:
With other maintainers, we have thought that NativeLBByDefault was a good enough name because it's homogeneous with other existing options that allow the same kind of behavior (general option in the static configuration that can be overridden in the dynamic configuration), like [connectByDefault](https://doc.traefik.io/traefik/providers/consul-catalog/#connectbydefault) in Consul Catalog.

Reply:
Updated.